src/models/converters/safes.rs

```diff
@@ -1,4 +1,5 @@
-use crate::models::service::safes::{AddressEx, ImplementationVersionState, SafeInfoEx};
+use crate::models::service::addresses::AddressEx;
```

Comment:
This used to be next so `safes`, but it's cool that we create a module of its own

Reply:
I moved it out of Safes as it was used in a lot of places unrelated to the Safe model